init.lua

```diff
@@ -46,16 +46,12 @@ local csm_order = false
 -- Recalculate the CSM loading order
 -- TODO: Make this nicer
 local function recalc_csm_order()
-    local loaded = {}
     local staging = {}
     local order = {':init'}
     local unsatisfied = {}
     for name, def in pairs(sscsm.registered_csms) do
         assert(name == def.name)
-        if name:sub(1, 1) == ':' then
-            loaded[name] = true
-        elseif not def.depends or #def.depends == 0 then
-            loaded[name] = true
+        if not def.depends or #def.depends == 0 then
             table.insert(staging, name)
```

Comment:
This code is now executed when `name:sub(1, 1) == ':'` which will create unintended side-effects.